src/components/PoolingWidget/DefineSpread.styled.ts

```diff
@@ -15,6 +15,7 @@ export const DefineSpreadWrapper = styled(InputBox)`
     width: 100%;
     height: 5.6rem;
     padding: 0 1rem;
+    background: white;
```

Comment:
can you make this `var(...)` for dark mode

Reply:
Ah, sorry forgot to change, will do now.